Farm&Gun/Assets/Scripts/Entities/Opponents/Opponent.cs

```diff
@@ -4,20 +4,18 @@
 
 public class Opponent : MonoBehaviour
 {
-    [SerializeField]
-    private float power;
-
-    [SerializeField]
-    private float speed;
-
     [SerializeField]
     private float shotForce;
 
     [SerializeField]
     private NavMeshAgent agent;
 
+    private float buffAmount = 2;
```

Comment:
If it isn't changed anywhere - make it a Constant

Reply:
fixed